subjects/ascii-art/ascii-color.audit.en.md

```diff
@@ -1,20 +1,20 @@
-#### Ascii-Color
+#### Functional
 
-##### Try passing as arguments "hello world" --color=red.
+##### Try passing as arguments `""hello world" --color=red"`.
 ###### Does it displays the expected result?
-##### Try passing as arguments "1 + 1 = 2" --color=green.
+##### Try passing as arguments `""1 + 1 = 2" --color=green".
 ###### Does it display the expected result?
-##### Try passing as arguments "(%&) ??" --color=yellow.
+##### Try passing as arguments `""(%&) ??" --color=yellow"`.
```

Comment:
We are using the back quotes (``) to highlight the arguments. There is no need to add extra double quotes ("")